### PR TITLE
Formatting changes to follow tidyverse style guide

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
I have reverted some changes where `{styler}` couldn't preserve the deliberate alignments, or changed the positions of comments related to ignoring lint, etc.

### CODE REVIEW

File: R/exclude.R [hunk 1]
```diff
@@ -237,9 +236,10 @@ normalize_exclusions <- function(x, normalize_path = TRUE,
 
     if (any(bad)) {
       stop("Full file exclusions must be character vectors of length 1. items: ",
-           toString(which(bad)),
-           " are not!",
-           call. = FALSE)
+        toString(which(bad)),
```
Comment (MichaelChirico) [inline R/exclude.R:239]: I prefer the first argument bumped to a new kind as well

File: R/expect_lint.R [hunk 2]
```diff
@@ -57,13 +57,14 @@ expect_lint <- function(content, checks, ..., file = NULL, language = "en") {
...
 
-  wrong_number_fmt  <- "got %d lints instead of %d%s"
+  wrong_number_fmt <- "got %d lints instead of %d%s"
   if (is.null(checks)) {
     msg <- sprintf(wrong_number_fmt, n_lints, length(checks), lint_str)
     return(testthat::expect(n_lints %==% 0L, msg))
   }
 
-  if (!is.list(checks) || !is.null(names(checks))) { # vector or named list
+  if (!is.list(checks) || !is.null(names(checks))) {
```
Comment (MichaelChirico) [inline R/expect_lint.R:66]: is this a tidyverse rule?

File: R/get_source_expressions.R [hunk 3]
```diff
@@ -348,11 +353,12 @@ lint_parse_error_nonstandard <- function(e, source_expression) {
     if (grepl("attempt to use zero-length variable name", e$message, fixed = TRUE)) {
       # empty symbol: ``, ``(), ''(), ""(), fun(''=42), fun(""=42), fun(a=1,""=42)
       loc <- re_matches(source_expression$content,
-                        rex("``" %or%
-                              list(or("''", '""'), any_spaces, "(") %or%
-                              list(or("(", ","), any_spaces, or("''", '""'), any_spaces, "=")),
-                        options = "multi-line",
-                        locations = TRUE)
+        rex("``" %or%
```
Comment (MichaelChirico) [inline R/get_source_expressions.R:356]: bump the first argument too please

File: R/get_source_expressions.R [hunk 4]
```diff
@@ -388,20 +394,29 @@ lint_parse_error_nonstandard <- function(e, source_expression) {
...
+      capture(
+        name = "line",
+        digits
+      ),
       ":",
-      capture(name = "column",
-        digits),
+      capture(
+        name = "column",
+        digits
```
Comment (MichaelChirico) [inline R/get_source_expressions.R:409]: ditto

File: R/get_source_expressions.R [hunk 5]
```diff
@@ -388,20 +394,29 @@ lint_parse_error_nonstandard <- function(e, source_expression) {
...
+      ),
       ":",
       space,
-      capture(name = "message",
-        anything),
-      "\n")
+      capture(
+        name = "message",
```
Comment (MichaelChirico) [inline R/get_source_expressions.R:414]: ditto

File: R/methods.R [hunk 6]
```diff
@@ -11,34 +10,36 @@ print.lint <- function(x, ...) {
   cat(
     sep = "",
     crayon::bold(x$filename, ":",
-    as.character(x$line_number), ":",
-    as.character(x$column_number), ": ", sep = ""),
+      as.character(x$line_number), ":",
+      as.character(x$column_number), ": ",
+      sep = ""
```
Comment (MichaelChirico) [inline R/methods.R:15]: is sep= used twice here?

File: R/exclude.R (hunk 1 above)
Comment (IndrajeetPatil) [reply R/exclude.R:239]: Sorry, not sure what you mean by "bump" here.

Currently, it is:

```r
    if (any(bad)) {
      stop("Full file exclusions must be character vectors of length 1. items: ",
        toString(which(bad)),
        " are not!",
        call. = FALSE
      )
    }
```

And you'd like it to be formatted as the following?

```r
    if (any(bad)) {
      stop(
        "Full file exclusions must be character vectors of length 1. items: ",
        toString(which(bad)),
        " are not!",
        call. = FALSE
      )
    }
```

File: R/utils.R [hunk 7]
```diff
@@ -183,15 +183,17 @@ Linter <- function(fun, name = linter_auto_name()) { # nolint: object_name.
...
   terminal_newline <- TRUE
-  lines <- withCallingHandlers({
-    readLines(file, warn = TRUE, ...)
-  },
-  warning = function(w) {
-    if (grepl("incomplete final line found on", w$message, fixed = TRUE)) {
-      terminal_newline <<- FALSE
-      invokeRestart("muffleWarning")
+  lines <- withCallingHandlers(
+    {
```
Comment (MichaelChirico) [inline R/utils.R:187]: I'd drop the braces

File: R/expect_lint.R (hunk 2 above)
Comment (IndrajeetPatil) [reply R/expect_lint.R:66]: I don't think so. I will revert this change; definitely unwanted for our purposes.

File: R/exclude.R (hunk 1 above)
Comment (MichaelChirico) [reply R/exclude.R:239]: yep that's what I mean

File: R/expect_lint.R (hunk 2 above)
Comment (MichaelChirico) [reply R/expect_lint.R:66]: is there a styler bug to report?

File: R/methods.R (hunk 6 above)
Comment (IndrajeetPatil) [reply R/methods.R:15]: Btw, I am laughing my ass off at this comment  🤣

```r
# swap tabs for spaces for #528 (sorry Richard Hendricks)
```

File: R/expect_lint.R (hunk 2 above)
Comment (IndrajeetPatil) [reply R/expect_lint.R:66]: Yes, this actually messes with lintr directives in comments.

Here is a minimal and compelling reprex:

``` r
styler::style_text(
  "T_and_F_symbol_linter <- function() { # nolint: object_name.
  }"
)
#> T_and_F_symbol_linter <- function() {
#>   # nolint: object_name.
#> }
```

<sup>Created on 2022-09-19 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

File: R/methods.R (hunk 6 above)
Comment (MichaelChirico) [reply R/methods.R:16]: oh, I see it more clearly now that the blear's gone, thanks!

**Review by MichaelChirico — APPROVED** [review]

File: R/expect_lint.R (hunk 2 above)
Comment (IndrajeetPatil) [reply R/expect_lint.R]: Would you like to create an issue or should I?

Comment (MichaelChirico) [reply R/expect_lint.R]: go ahead since you've got the reprex already, thanks

Comment (IndrajeetPatil) [reply R/expect_lint.R]: Done: https://github.com/r-lib/styler/issues/1004

**Pull request merged**